content/en/synthetics/private_locations/_index.md

```diff
@@ -531,32 +531,60 @@ You can then go to any of your API or Browser test creation form, and tick your
+* Additional requirements based on the type of tests run by the private location:
 
-#### Disk
+| Test type                                     | CPU/Memory/Disk recommendation    |
+| --------------------------------------------- | --------------------------------- |
+| [API tests][12] and [Multistep API tests][13] | 20mCores/5MiB/1MiB per test run   |
+| [Browser tests][14]                           | 150mCores/1GiB/10MiB per test run |
 
-The recommendation for disk size is to allocate ~ 10MiB/slot (1MiB/slot for API-only private locations).
+**Example:** For a private location running only Browser tests, with a maximum number of concurrent test runs of `10`, the recommendation for a safe usage is 
+~ 1.5Core CPU `(150mCores + (150mCores*10 test runs))`, ~ 10GiB memory `(150MiB + (1GiB*10 test runs))`, and ~ 100MiB disk `(10MiB*10 test runs)`.
+
+**Note:** Resources requirements may vary based on the application being tested (size/number of assets to be loaded, etc.).
+
+**Note:** When running both API/Multistep API tests and Browser tests on a single private location, the recommendation is to perform computation using Browser tests resource requirements.
+
+### Assign resources to your private location
+
+Once you know about the [**total** requirements for your private location](#private-location-total-hardware-requirements), you can decide how you want these resources to be distributed:
+
+* You can assign all resources to a single worker (ie. to a single container). In this case:
+  * Set the [`concurrency` parameter][15] to `maximum number of test runs that can be executed in parallel on your private location`.
+  * Assign your [total private location resource requirements](#private-location-total-hardware-requirements) to your unique worker.
+* You can distribute resources across several workers. You can indeed run several containers for one private location with one single configuration file. In this case:
+  * Set the [`concurrency` parameter][15] to `maximum number of test runs that can be executed on your private location / number of workers associated with your private location`. Each worker will then request `N` tests to run depending on its [`concurrency` parameter][15] and number of free slots: when worker 1 is processing tests, worker 2 requests the following tests, etc.
+  * Assign `total private location resource requirements / number of workers` resources to each private location worker.
+
+**Example:** For a private location running only Browser tests, with a maximum number of concurrent test runs of `10`, your private location requires ~ 1.5 core CPU, ~ 10GiB memory, and ~ 100MiB disk. If you want to distribute these resources across 2 workers, the [`concurrency` parameter][15] should be set to `5`, and each worker should be allocated ~ 750mCores CPU, ~ 5GiB memory, and ~ 50MiB disk.
```

Comment:
Should we commit to say that this should spread the load of the PL?

Reply:
Added a note on that earlier in the section